src/test/java/graphql/annotations/GraphQLDataFetcherTest.java

```diff
@@ -14,17 +14,13 @@
  */
 package graphql.annotations;
 
-import java.util.HashMap;
-
 import graphql.ExecutionResult;
 import graphql.GraphQL;
-import graphql.schema.DataFetcher;
-import graphql.schema.DataFetchingEnvironment;
-import graphql.schema.GraphQLObjectType;
-import graphql.schema.GraphQLSchema;
-import graphql.schema.PropertyDataFetcher;
+import graphql.schema.*;
```

Comment:
Can you please use direct imports.  If you are using IDEA, set your min imports to 9999